### PR TITLE
Add info about min docker version. Update sanity check.

### PR DESCRIPTION
**Pull request by IgorSusmelj**
## Changes

Only minor changes of the docs

- Add info about min docker version
- Update sanity check

image

closes LIG-947

### CODE REVIEW

File: docs/source/docker/advanced/active_learning.rst [hunk 1]
```diff
@@ -11,6 +11,10 @@ about the uncertainty of the model for the given sample.
 Learn more about the concept of active learning scores:
 :ref:`lightly-active-learning-scorers`.
 
+.. warning:: Note that active learning requires a minimum
```
Comment (japrescott) [inline docs/source/docker/advanced/active_learning.rst:14]: I think a warning msg is a bit heavy. Maybe an info box is enough?

File: docs/source/docker/advanced/datasource_predictions.rst [hunk 2]
```diff
@@ -13,6 +13,9 @@ add their predictions to the datasource.
 If you already have labels instead of predictions, you can treat them
 just as predictions and upload them the same way.
 
+.. warning:: Note that using predictions requires a minimum 
+    docker of version 2.2. You can check your docker version by running the
```
Comment (japrescott) [inline docs/source/docker/advanced/datasource_predictions.rst:17]: its not called "docker" :)
Note that using predictions requires the worker docker image of version 2.2.0 or higher

**Review by japrescott — CHANGES_REQUESTED** [review]

File: docs/source/docker/advanced/datasource_predictions.rst [hunk 3]
```diff
@@ -13,8 +13,8 @@ add their predictions to the datasource.
 If you already have labels instead of predictions, you can treat them
 just as predictions and upload them the same way.
 
-.. warning:: Note that using predictions requires a minimum 
-    docker of version 2.2. You can check your docker version by running the 
+.. note:: Note that the object level features require a minimum
```
Comment (japrescott) [inline docs/source/docker/advanced/datasource_predictions.rst:16]: predictions

File: docs/source/docker/advanced/datasource_predictions.rst [hunk 4]
```diff
@@ -13,8 +13,8 @@ add their predictions to the datasource.
 If you already have labels instead of predictions, you can treat them
 just as predictions and upload them the same way.
 
-.. warning:: Note that using predictions requires a minimum 
-    docker of version 2.2. You can check your docker version by running the 
+.. note:: Note that the object level features require a minimum 
+    Lightly Worker of version 2.2. You can check your docker version by running the
```
Comment (japrescott) [inline docs/source/docker/advanced/datasource_predictions.rst:17]: you can chekc your docker version -> you can check the version of Lightly Worker by running the sanity check..

**Review by japrescott — CHANGES_REQUESTED** [review]

File: docs/source/docker/advanced/datasource_predictions.rst (hunk 3 above)
Comment (IgorSusmelj) [reply docs/source/docker/advanced/datasource_predictions.rst:16]: oh, yes I messed things up with copy paste :(

**Review by japrescott — APPROVED** [review]

**Pull request merged**